src/components/ImageComponents/Image.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import "./Image.style.css";
+
+const Image = ({ url }) => {
+  return <img src={url} alt="img" className="img" />;
```

Comment:
`alt` should be passed as a prop as well. Also, avoid putting text like 'image', 'photo' or anything like that inside the `alt`, because screen readers already read img alt with as a photo